deltachat-ios/View/ContactCell.swift

```diff
@@ -231,7 +231,7 @@ class ContactCell: UITableViewCell {
     }
 
     // use this update-method to update cell in cellForRowAt whenever it is possible - other set-methods will be set private in progress
-    func updateCell(dcContext: DcContext, cellViewModel: AvatarCellViewModel) {
+    func updateCell(cellViewModel: AvatarCellViewModel) {
```

Comment:
Yeah!

We could also consider adding the needed DcChat-object to the cellViewModel.cellData, in case we want to avoid accessing DcContext.shared here.  Maybe in another PR.